src/ape_plugins/_cli.py

```diff
@@ -22,38 +22,89 @@ def cli():
     """
 
 
-@cli.command(name="list", short_help="List installed plugins")
+def _remove_prefix_add_version(set_convert, _prefix, version):
+    output = set()
+    text = None
+    if set_convert:
```

Comment:
This should never be `None` in my opinion. If the set is `None`, there is no reason to call this method